ime/app/src/main/java/com/anysoftkeyboard/ime/AnySoftKeyboardSuggestions.java

```diff
@@ -1461,6 +1468,17 @@ public void onRemoved() {
             mRootView.removeCallbacks(mReHideTextAction);
         }
 
+        public void setCloseIconDrawable(Drawable closeIcon) {
+            mCloseIcon = closeIcon;
+        }
+
+        private void setCloseIcon() {
+            ImageView closeIcon = (ImageView) mRootView.findViewById(R.id.close_suggestions_strip_icon);
+            if (mCloseIcon != null) {
```

Comment:
why would it be `null`?

Reply:
I cleaned up some stuff.  Can you review again?